src/cases/containers/case-list/case-list.component.spec.ts

```diff
@@ -350,6 +350,32 @@ describe('CaseListComponent', () => {
     });
   });
 
+  describe('Should see cases unselectable information', () => {
+    it('should see why are some cases unselectable', () => {
+      const resultView = {
+        columns: [],
+        results: [
+          {
+            case_id: 'case_123',
+          }
+        ],
+        result_error: null
+      };
+      component.onResultsViewHandler(resultView);
+      expect(component.hasResults()).toBeTruthy();
+    });
+
+    it('should not see why are some cases unselectable', () => {
+      const resultView = {
+        columns: [],
+        results: [],
+        result_error: null
+      };
+      component.onResultsViewHandler(resultView);
+      expect(component.hasResults()).toBeFalsy();
```

Comment:
Validation to check "unselectable information" not displayed in html is missing.